src/pages/explore/masternodes/SvgIconsColumn.tsx

```diff
@@ -44,18 +48,22 @@ export function SvgIconsColumn({
   customSectionStyle,
   customClassStyle,
   customParagraphStyle,
+  customDescStyle,
+  isCenterAligned,
 }: {
   items: Array<BlockchainFeatureItemProp>;
   id: string;
   customSectionStyle?: string;
   customClassStyle?: string;
   customParagraphStyle?: string;
+  customDescStyle?: string;
+  isCenterAligned?: boolean;
 }): JSX.Element {
   return (
     <div
       className={classNames(
         customSectionStyle,
-        "flex flex-1 flex-wrap justify-between md:gap-y-[120px] lg:mr-[64px] lg:gap-y-16 lg:gap-x-10"
+        "flex flex-1 flex-wrap justify-between gap-y-12 md:gap-y-[120px] lg:gap-y-16 lg:gap-x-10"
```

Comment:
is there any regression for adding the new spacing here? since it is used in quite many components

Reply:
donee!! Checked through all the `SvgIconsColumn` instances